app/models/budget/investment.rb

```diff
@@ -252,5 +258,18 @@ def set_denormalized_ids
         self.budget_id ||= self.heading.try(:group).try(:budget_id)
       end
 
+      def check_image_dimensions
+        temp_file = image.queued_for_write[:original]
+        unless temp_file.nil?
+          dimensions = Paperclip::Geometry.from_file(temp_file)
+          width = dimensions.width
+          height = dimensions.height
+
+          if width < 475 || height < 475
```

Comment:
Maybe a class constant `MAX_SIZE = 475` could be handy here as well as on line 14 `... large: "x#{MAX_SIZE}"...` To make it easier to maintain in the future or by forks. What do you think? 

Also we could dispense `width` and `height` variables as both are only used once, and go with `if dimensions.width < MAX_SIZE || dimensions.height < MAX SIZE` right?